[4519] Nextify out of box blueprints - Update files

jvega190 opened the pull request:
https://github.com/craftercms/craftercms/issues/4519

jvega190 committed:
Add pending files

jvega190 committed:
Add missing groovy files

jvega190 committed:
Added missing content

rart commented on src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/site/taxonomy/categories.xml:
```diff
@@ -1,10 +1,11 @@
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
   ~
```
Please use right copy right. Year needs fixing and I think this is the previous copy text

rart reviewed (changes requested):
Bad copyright in several places.

jvega190 committed:
Update copyright

jvega190 committed:
Add scss sources

jvega190 committed:
Fix merge problems

jvega190 commented:
@rart here's the list of changes:
- Updated copyrights where needed
- Added the articles in proper new folders (2020, 2021)
- Added scss sources
- Added new font files (font awesome 5)
- Fixed merge problems (content, form-definitions and master branch newer commits)
